message/messaging_service.hh

```diff
@@ -202,7 +202,9 @@ enum class messaging_verb : int32_t {
     TASKS_GET_CHILDREN = 74,
     TABLET_REPAIR = 75,
     TRUNCATE_WITH_TABLETS = 76,
-    LAST = 77,
+    BUILD_VIEWS_RANGE = 77,
+    ABORT_VBC_WORK = 78,
+    LAST = 79,
 };
```

Comment:
File seems too short. Where are the implementations?

Reply:
In the IDL.